### PR TITLE
[NUI] Add Seamless transition

### PR DESCRIPTION
**Pull request by huiyueun**
Add NUIFrameBroker & NUIFrameProvider

Signed-off-by: huiyu.eun <huiyu.eun@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Interop/Interop.Libraries.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class Libraries
+        {
+            public const string FrameBroker = "libframe-broker.so.1";
```
Comment (flyofsky) [inline src/Tizen.NUI/src/internal/Interop/Interop.Libraries.cs:23]: When the class is internal, does the property need to be public?

File: src/Tizen.NUI/src/public/FrameBroker/FrameBrokerBase.cs [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    /// <summary>
+    /// Represents the Frame Broker.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class FrameBrokerBase : IDisposable
+    {
+        private string LogTag = "Tizen.NUI";
+        private readonly SafeFrameBrokerHandle _handle;
+        private static Dictionary<int, Interop.FrameBroker.AppControlResultCallback> _resultCallbackMaps = new Dictionary<int, Interop.FrameBroker.AppControlResultCallback>();
```
Comment (flyofsky) [inline src/Tizen.NUI/src/public/FrameBroker/FrameBrokerBase.cs:35]: Is there any reason for being private but static?

Comment (huiyueun) [reply src/Tizen.NUI/src/public/FrameBroker/FrameBrokerBase.cs:35]: fixed

File: src/Tizen.NUI/src/internal/Interop/Interop.Libraries.cs (hunk 1 above)
Comment (huiyueun) [reply src/Tizen.NUI/src/internal/Interop/Interop.Libraries.cs:23]: fixed

**Pull request closed without merging**